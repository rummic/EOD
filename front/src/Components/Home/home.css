body{
    width: 100%;
}

.homeb{
    width:100%;
}

.homebox{
    margin: auto;
    width:100%;
    background-color: white;
    position: fixed;
    height: 100%;
    margin-left: 20%;
}

.homebox .homebox-options{
    width: 70%;
    position: fixed;
    height: 80%;
    margin-left: 5%;
    margin-top: 50px;

}


.homebox .homebox-options .homebox-option{
    float: left;
    width: calc((100% / 2) - 20px);
    height: 400px;
    margin: 10px;
    margin-top: 10%;
    border-radius: 20px;
    margin-bottom: 30px;
    text-align: center;
    background-color: rgba(42, 26, 184, 0.877);
    box-shadow: inset 0px 0px 108px -1px rgba(5,0,105,1);

    
}


.homebox .homebox-options .homebox-option .homebox-option-content{
    display: table;
    height: 100%;
    width: 100%;
    color:white;
    font-size: 40px;
    font-weight: 700;
    font-family: 'Times New Roman', Times, serif;
}

.homebox .homebox-options .homebox-option .homebox-option-content p{
    display: table-cell;
    vertical-align: middle;
    height: 100%;
    width: 100%;
}








