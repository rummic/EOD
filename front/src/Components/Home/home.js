import React, { Component } from "react";
import { GoFile, GoFileSymlinkFile } from "react-icons/go";
import Sidebar from "../Navbar/sidebar";
import { Redirect, Link } from "react-router-dom";
import "./home.css";

class home extends Component {
  render() {
    if (!sessionStorage.getItem("token")) {
      return <Redirect to={"/login"} />;
    }
    return (
      <div className="homeb">
        <Sidebar history={this.props.history} />
        <div className="homebox">
          <div className="homebox-options">
            <Link to="/userdocument">
              <div className="homebox-option">
                <div className="homebox-option-content">
                  <div className="homebox-option-content-con">
                    <GoFile className="option-image" />
                    <p>Zobacz dokumenty</p>
                  </div>
                </div>
              </div>
            </Link>
            <Link to="/addfile">
              <div className="homebox-option">
                <div className="homebox-option-content">
                  <div className="homebox-option-content-con">
                    <GoFileSymlinkFile className="option-image" />
                    <p>Dodaj dokumenty</p>
                  </div>
                </div>
              </div>
            </Link>
          </div>
        </div>
      </div>
    );
  }
}

export default home;
