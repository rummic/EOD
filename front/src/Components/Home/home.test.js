import React from 'react';
import { shallow } from 'enzyme';
import home from './home.js';

describe('home', () => {
    test('Rendering home component', () => {
        const wrapper = shallow(<home />);
        expect(wrapper).toMatchSnapshot();
    });
});  