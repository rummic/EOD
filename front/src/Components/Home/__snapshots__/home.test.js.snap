// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home Rendering home component 1`] = `ShallowWrapper {}`;
